docs/rfcs/2022-12-09-ephemeral-conversations.md

```diff
@@ -0,0 +1,208 @@
+# Ephemeral conversations with existing contacts
+
+Ephemeral conversation inside existing conversation with stricter security properties.
+
+- additional level of encryption for message bodies & text, chat items content & text (more?) inside chat db
```

Comment:
text is enough

Reply:
and whatever will be there in the message. The open question is what to do with files.